modules/jaggery-apps/app-publisher-web/apis/v1/entitlement_api_router.jag

```diff
@@ -50,6 +52,12 @@ require('/modules/publisher.js').exec(function(ctx) {
         print(partialList);
     });
 
+    routeManager.register('GET', 'publisher', '/publisher/api/entitlement/get/policy/Group/by/appId/{applicationId}', function (context) {
```

Comment:
There was an issue with the url router. (if there are more than one url with same no of parameters, it referred the first url)
So till we fix the router issue I had to use this pattern to make the code work.